zulipterminal/cli/run.py

```diff
@@ -387,15 +398,21 @@ def check_for_default_zuliprc() -> str:
     return ""
 
 
-def parse_zuliprc(zuliprc_str: str) -> Tuple[Dict[str, SettingData], str]:
+def parse_zuliprc(
+    zuliprc_str: str, new_realm: bool
+) -> Tuple[Dict[str, SettingData], str]:
     zuliprc_path = (
-        check_for_default_zuliprc()
-        if zuliprc_str == ""
-        else path.expanduser(zuliprc_str)
+        (
+            check_for_default_zuliprc()
+            if zuliprc_str == ""
+            else path.expanduser(zuliprc_str)
+        )
+        if not new_realm
+        else ""
     )
     while zuliprc_path == "" or not path.exists(zuliprc_path):
         try:
-            zuliprc_path = fetch_zuliprc(zuliprc_path)
+            zuliprc_path = fetch_zuliprc(zuliprc_path, new_realm)
```

Comment:
Re. naming,
used 'realm' for all internal references,
and 'org' / 'organization' for all references visible to the user.